README.md

```diff
@@ -1605,4 +1605,477 @@ Other Style Guides
 
 **[⬆ back to top](#table-of-contents)**
 
+
+## Commas
+
+  - [19.1](#19.1) <a name='19.1'></a> Leading commas: **Nope.**
+
+    ```javascript
+    // bad
+    const story = [
+        once
+      , upon
+      , aTime
+    ];
+
+    // good
+    const story = [
+      once,
+      upon,
+      aTime,
+    ];
+
+    // bad
+    const hero = {
+        firstName: 'Ada'
+      , lastName: 'Lovelace'
+      , birthYear: 1815
+      , superPower: 'computers'
+    };
+
+    // good
+    const hero = {
+      firstName: 'Ada',
+      lastName: 'Lovelace',
+      birthYear: 1815,
+      superPower: 'computers',
+    };
+    ```
+
+  - [19.2](#19.2) <a name='19.2'></a> Additional trailing comma: **Yup.**
```

Comment:
I like trailing commas, especially since we're matching the case here using Babel everywhere.
However, let's make sure to also update your `.eslintrc`s